.gitignore

```diff
@@ -32,3 +32,7 @@ test/test_schema.db
 *test_schema.db
 .idea
 /Pipfile*
+/.pytest_cache/
```

Comment:
**mike bayer** (zzzeek) wrote:

pet peeve, can we have various .gitignores be in a separate commit ?

Reply:
ok l'll move it to a new pr